jquery.flot.logaxis.js

```diff
@@ -74,6 +74,8 @@ Set axis.mode to "log" to enable.
 
             if (max < min) {
                 max = axis.max = axis.datamax | axis.options.max;
```

Comment:
The | operator is working on bits. Is this really working as supposed to?

Is it possible to write a test for this case? I'm sure someone will end up breaking something here again sooner or later.